src/robusta/integrations/mattermost/sender.py

```diff
@@ -144,7 +147,10 @@ def send_finding_to_mattermost(self, finding: Finding, platform_enabled: bool):
             blocks.append(MarkdownBlock(finding.description))
 
         for enrichment in finding.enrichments:
-            blocks.extend(enrichment.blocks)
+            if enrichment.annotations.get(EnrichmentAnnotation.SCAN, False):
```

Comment:
I think here it should be:
```
            if enrichment.annotations.get(EnrichmentAnnotation.SCAN, False):
                enrichment.blocks = [Transformer.scanReportBlock_to_fileblock(b) for b in enrichment.blocks]
            
            blocks.extend(enrichment.blocks)
```

The `blocks` array is the collection sent to mattermost later on, so we want to add the converted file to it

Reply:
You're right. Got it!